ArraysAmbiguousSNPMiscallingMar2019/allele_flip/README.md

```diff
@@ -0,0 +1,46 @@
+This subdirectory provides tools for assessing whether your PLINK format genotyping data is affected by the strand ambiguous bug, and for fixing the bug through flipping DNA strand.
+
+## Assessing Strand Ambiguous Bug
+The shell script `compare_frequencies_against_1kg.r` can be used to determine if the bug has affected your data by plotting allele frequencies against 1000 genomes allele frequencies. The file `af.pdf` is an example output. 
```

Comment:
do we need to keep both pre_af.pdf and af.pdf here?

Reply:
For now I generated AF plots for pre-flipped data and post-flipped data, named as pre_flip.pdf and post_flip.pdf respectively. Do you think it would be better to name them as pre_af.pdf and  af.pdf?